lib/rucio/core/config.py

```diff
@@ -23,6 +23,7 @@
 
```

Comment:
looks good to me, don't forget to add yourself to the authorlist at the top

Reply:
Done, updated the MR.